### PR TITLE
Refactor ServiceRequest.js

### PR DESCRIPTION
**Pull request by planigan**
I pulled out all the data for the single and multi line fields and refactored the markup to map over it. If you don't like the hyphens in the property names, or want to format them in some other way, they are all using a new formatLabelToProperty function, so you can just change it there. I initially was trying to use object spread to set the initial state, but for some reason, it wasn't updating with the default falsey values set, so I resorted to the old Object.assign. Oddly, object spread works fine in the reduce functions I wrote.

### CODE REVIEW

**Comment by JesseRWeigel** [comment]
Thanks @planigan !

File: src/views/ServiceRequest.js [hunk 1]
```diff
@@ -32,15 +32,63 @@ const styles = {
...
+    const stringProps = [
+      ...this.singleLineFields,
+      ...this.multiLineFields
+    ].reduce(
+      (acc, label) => ({
+        [this.formatLabelToProperty(label)]: '',
+        ...acc
+      }),
+      {}
+    )
```
Comment (planigan) [inline src/views/ServiceRequest.js]: You seemed like you didn't really understand these reduce functions today, so, I thought I would take a stab at explaining. 

A basic `Array#reduce` takes a function and a starting value, in this case, the empty object `{}`. The function gets passed an accumulator (acc in the above code) and the current element of the Array (label in this case) as the first two arguments. This is just using computed properties to format them from the list of fields and set them to an empty string and using object spread (`...acc`) to add the values that have been accumulated. This is all operating over the temporary array that is spreading the two arrays into itself. Once it has looped over all the values in singleLineFields and multiLineFields we are left with stringProps which is an object that contains all the properties we want to put in the state with an empty string initial value.

So a simpler use of reduce might look like this:
```js
[2,2,2,2,2].reduce( (acc, val) => acc + val, 0 ) // returns 10
```

File: src/views/ServiceRequest.js [hunk 2]
```diff
@@ -32,15 +32,63 @@ const styles = {
...
     this.state = {
-      fileInput: null,
-      name: '',
-      email: '',
-      phone: ''
+      fileInput: null
     }
+    Object.assign(this.state, stringProps, checkboxProps)
```
Comment (planigan) [inline src/views/ServiceRequest.js]: Now that we have our objects that contain our desired properties (stringProps and checkboxProps), we can add them to the state with `Object.assign` which just adds the properties from the `2..n` arguments to the object passed as the 1st argument.

File: src/views/ServiceRequest.js (hunk 1 above)
Comment (JesseRWeigel) [inline src/views/ServiceRequest.js]: Yes, this helps a lot. Thank you!

**Pull request merged**